Sources/Bot/Services/MergeService.swift

```diff
@@ -55,6 +55,7 @@ public final class MergeService {
             }
 
         gitHubEvents.events
+            .observe(on: scheduler)
```

Comment:
We should offload this to our internal scheduler without blocking the source of this events.